packages/react-composites/src/composites/ChatComposite/image-upload/useImageUpload.tsx

```diff
@@ -47,6 +47,24 @@ const getInlineImageData = async (image: string): Promise<Blob | undefined> => {
   return blobImage;
 };
 
+/* @conditional-compile-remove(rich-text-editor-image-upload) */
+/**
+ * @internal
+ * Exported only for unit testing
+ */
+export const removeImageTags = (event: { content: DocumentFragment }): void => {
```

Comment:
@Leah-Xia-Microsoft let's move it to utils file instead of having it in the `useImageUpload` hook file